src/Symfony/Bridge/Doctrine/Messenger/DoctrinePingConnectionMiddleware.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
+use Symfony\Component\Messenger\Middleware\StackInterface;
+
+/**
+ * Wraps all handlers in a single doctrine transaction.
+ *
+ * @author Tobias Nyholm <insidestyles@gmail.com>
+ *
+ * Class DoctrinePingConnectionMiddleware
+ */
+class DoctrinePingConnectionMiddleware implements MiddlewareInterface
+{
+    /** @var ManagerRegistry */
+    private $managerRegistry;
+
+    /** @var string */
+    private $entityManagerName;
+
+    /** @var bool */
+    private $forceCloseConnection = false;
+
+    public function __construct(
```

Comment:
Symfony uses long lines, please change the CS accordingly

Reply:
curl https://fabbot.io/patch/symfony/symfony/31061/11c4a4b182b6029ee637d761b7f2f6339f51bc3e/cs.diff | patch -p0

updated. but this does not fix if arguments are in separated lines